src/static/css/mobile-carousel.less

```diff
@@ -0,0 +1,113 @@
+/* ==========================================================================
+   cfgov-refresh
+   Overrides for slick carousel on /the-bureau/ and /careers/working-at-cfpb/.
+   ========================================================================== */
+
+.js-mobile-carousel {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/d889d89bc1b3711c4234fb5929d913b513cbf980